Resources/Prototypes/GameRules/events.yml

```
@@ -448,9 +463,9 @@
   categories: [ HideSpawnMenu ]
   components:
   - type: StationEvent
-    earliestStart: 50
-    minimumPlayers: 30
-    weight: 2
+    earliestStart: 90
+    minimumPlayers: 40
+    weight: 1 # Zombies was happening basically every single survival round, so now it's super rare
```

YAMLlint (1.35.1)

[warning] too few spaces before comment

(comments)